src/lib/widgets/now-playing/ui/specs/now-playing.spec.ts

```diff
@@ -0,0 +1,5 @@
+export {};
```

Comment:
issue: you should also probably test that the `EpisodeDisplay` and `FullControls` are integrated correctly

suggestion: add a couple more tests that check that the episode title is present somewhere `inTheDocument` and the same for a button that says "Play"